src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,9 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: 'Under construction',
+  reducers: {},
+});
```

Comment:
- You have done great so far :+1: but there are a few requirements you still need to implement before moving to the next milestone. :+1: :100: 
- [ ] Looking at the project requirements, the `categoriesSlice` is meant to have one reducer `checkStatus`. you haven't implemented any reducer in the categoriesSlice, I suggest you do so. :small_airplane: :smile: 

- - Screenshot reference:
![Selection_018](https://github.com/jcmunav63/bookstore/assets/107632780/7e2bfcf2-4c13-4c51-a0d5-6e7c9e4cad5a)